dependencies.props

```diff
@@ -38,16 +38,16 @@
       <CurrentRef>$(CoreFxCurrentRef)</CurrentRef>
     </RemoteDependencyBuildInfo>
     <RemoteDependencyBuildInfo Include="CoreClr">
-      <BuildInfoPath>$(BaseDotNetBuildInfo)coreclr/$(DependencyBranch)</BuildInfoPath>
-      <CurrentRef>$(CoreClrCurrentRef)</CurrentRef>
+      <!-- TODO: Revert RawUrl change when CoreCLR has a new 2.0.0 build. -->
+      <RawUrl>https://raw.githubusercontent.com/dotnet/versions/$(CoreClrCurrentRef)/$(BaseDotNetBuildInfo)coreclr/master</RawUrl>
     </RemoteDependencyBuildInfo>
     <RemoteDependencyBuildInfo Include="Standard">
       <BuildInfoPath>$(BaseDotNetBuildInfo)standard/$(DependencyBranch)</BuildInfoPath>
       <CurrentRef>$(StandardCurrentRef)</CurrentRef>
     </RemoteDependencyBuildInfo>
     <RemoteDependencyBuildInfo Include="WCF">
-      <BuildInfoPath>$(BaseDotNetBuildInfo)wcf/$(DependencyBranch)</BuildInfoPath>
-      <CurrentRef>$(WCFCurrentRef)</CurrentRef>
+      <!-- TODO: Revert RawUrl change when WCF has a new 2.0.0 build. -->
```

Comment:
Should be nothing blocking, but we didn't set up automatic builds for release/2.0.0 branch. @StephenBonikowsky, is there a button you need to click?